modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/ExecutionServiceImplTest.java

```diff
@@ -170,6 +172,14 @@ public void tearDown() throws Exception {
         }
 
         mailboxes.clear();
+
+        executers.forEach(executer -> {
```

Comment:
should we clean up the list of executors after each of tests?

Reply:
such a stupid bug... yes, we definitely should clean up this collection, thanks!

fixed